homeassistant/components/tuya/sensor.py

```diff
@@ -517,6 +517,18 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        # Total seconds of irrigation. Read-write value; the device appears to ignore the write action (maybe firmware bug)
+        TuyaSensorEntityDescription(
+            key=DPCode.TIME_USE,
+            name="Total watering time",
```

Comment:
Add a translation key, and move `name` into the translations file.

Reply:
Now its using the translation_key